internal/signerconfig/signerconfig.go

```diff
@@ -0,0 +1,75 @@
+// Copyright © 2022 Kaleido, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package signerconfig
+
+import (
+	"github.com/hyperledger/firefly/pkg/config"
+	"github.com/hyperledger/firefly/pkg/httpserver"
+	"github.com/hyperledger/firefly/pkg/wsclient"
+	"github.com/spf13/viper"
+)
+
+var ffc = config.AddRootKey
+
+var (
+	// BackendChainID optionally set the Chain ID manually (usually queries network ID)
+	BackendChainID = ffc("backend.chainId")
+	// FileWalletEnabled if the Keystore V3 wallet is enabled
+	FileWalletEnabled = ffc("fileWallet.enabled")
+	// FileWalletPath the path of the Keystore V3 wallet path
+	FileWalletPath = ffc("fileWallet.path")
+	// FileWalletFilenamesWith0xPrefix extension to append to the "from" address string to find the file (see metadata section for file types). All filenames must be lower case on disk.
```

Comment:
I think this description is wrong (actually configures whether or not to expect the 0x prefix on filenames)

Reply:
sorry - my english was trying to say that, I'll try to simplify per your suggestion

Reply:
ah, yeah this was copy/paste error